CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+##[2.7.0] - 2017-02-16
```

Comment:
Is this version somewhere else? How you figured out it was 2.6.3 ? We should bump it at that place too. Possibly add it to composer.json or wherever PHP package manager expects it.